php/razorpay_js/app.js

```diff
@@ -0,0 +1,138 @@
+/**
+*
+* This is a basic node js server Created
+* for showing a demo on how to implement razorpay
+* in chargebee application
+* 
+*/
+require('dotenv').config();
+const express = require('express'),
+    cors = require('cors'),
+    qs = require('qs'),
+    axios = require('axios');
+
+const app = express()
+app.use(cors())
+app.use(express.json());
+app.use(express.urlencoded());
+
+
+/**
+ * Env Variables
+ */
+ const CB_AUTH_TOKEN = Buffer.from(process.env.CB_API_KEY + ":").toString('base64');
+ const CB_AUTH_HEADER = `Basic ${CB_AUTH_TOKEN}`;
+
+ const RP_AUTH_TOKEN = Buffer.from(process.env.RAZORPAY_KEY + ":" + process.env.RAZORPAY_KEY_SECRET).toString('base64');
+ const RP_AUTH_HEADER = `Basic ${RP_AUTH_TOKEN}`;
+
+/**
+ * [ @Chargebee ]
+ * Used to create chargebee's estimation on subscription for items
```

Comment:
Used to create an estimate for a subscription(Product Catalog 2.0)
Refer to this [API](https://apidocs.chargebee.com/docs/api/estimates?prod_cat_ver=2&lang=curl#estimate_for_creating_a_subscription).